src/main/java/org/frc5687/steamworks/protobot/Constants.java

```diff
@@ -1,49 +1,60 @@
 package org.frc5687.steamworks.protobot;
 
-import org.frc5687.steamworks.protobot.Utils.Gamepad;
-
 /**
- * Created by Ben Bernard on 1/12/2017.
+ * Created by Admin on 1/16/2017.
  */
 public class Constants {
 
+    public static final int CYCLES_PER_SECOND = 50;
+
     public class Deadbands {
         /**
          * Deadband threshold for drive joysticks
          */
         public static final double DRIVE_STICK = 0.1;
     }
 
+    public class Limits {
+        /***
+         * Minimum time (in milliseconds) it should take to go from 0 to 1 (stop to full)
+         */
+        public static final double TIME_OF_ACCEL = 250;
+
+        /***
+         * Maximum accelerations per cycle
+         */
+        public static final double ACCELERATION_CAP = TIME_OF_ACCEL / CYCLES_PER_SECOND * 100;
+    }
     public class Calibration {
         /***
          * Controls the sensitivity algorithm.
          * 0 results in a linear control-to-speed relationship, while 1 results in cubed.
          *
          * NEVER SET ABOVE 1 OR BELOW 0
          */
-        public static final double SENSITIVITY_FACTOR = 0.5;
+        public static final double SENSITIVITY_FACTOR = .2;
     }
+
     public class Encoders {
         public class Defaults {
-            public static final boolean REVERSED = false;
-            public static final int SAMPLES_TO_AVERAGE = 20; //TODO Change pulse per rotation, samples to average, reversed, wheel diameter and max period to reflect new robot
+            public static final boolean REVERSED = false; //TODO change to new robot specifications
+            public static final int SAMPLES_TO_AVERAGE = 20;
             public static final int PULSES_PER_ROTATION = 1440;
             public static final double WHEEL_DIAMETER = 6;
             public static final double INCHES_PER_ROTATION = Math.PI * WHEEL_DIAMETER;
             public static final double SCALAR_RATIO = 8;
-            public static final double INCHES_PER_PULSE = INCHES_PER_ROTATION*SCALAR_RATIO/ PULSES_PER_ROTATION ;
+            public static final double INCHES_PER_PULSE = INCHES_PER_ROTATION * SCALAR_RATIO / PULSES_PER_ROTATION;
             public static final double MAX_PERIOD = 5;
         }
 
-        public class RightDrive {
-            public static final boolean REVERSED = Defaults.REVERSED;
-            public static final double INCHES_PER_PULSE = Defaults.INCHES_PER_PULSE;
-        }
 
-        public class LeftDrive {
-            public static final boolean REVERSED = true;
-            public static final double INCHES_PER_PULSE = Defaults.INCHES_PER_PULSE;
-        }
+    public class RightDrive {
```

Comment:
@TheMightyWarPig make sure the RightDrive class is indented properly under the Encoders class